saleor/product/models.py

```diff
@@ -453,6 +453,9 @@ def get_first_image(self) -> "ProductImage":
         images = list(self.images.all())
         return images[0] if images else self.product.get_first_image()
 
+    def get_ordering_queryset(self):
```

Comment:
Why do we add this function which is never called?

Reply:
See `SortableModel` abstract which the class uses now